rails-commenting.rb

```diff
@@ -6,25 +6,25 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) // This is the controller for the blog posts. 
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2) // An instance variable is created to store all of the blog posts.
     @posts = BlogPost.all
   end
 
-  # ---3)
+  # ---3) // This is the method that displays the blog post. It uses the find method to find the blog post by its unique ID from the postgreSQL database. The ID is passed in as a parameter.
   def show
     @post = BlogPost.find(params[:id])
   end
 
-  # ---4)
+  # ---4) // Creates a new blog post.
```

Comment:
What is unique about this method?

Reply:
It creates a form where you can submit the data. The create method on line 26 will push it to the database.